content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+
+
+
+
+## cloudify.gcp.nodes.TargetProxy
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A TargetHttpProxy or TargetHttpsProxy.
+
+Specify which using the `target_proxy_type` property.
+
+
+
+**Properties:**
+
+
+  * `ssl_certificate`
+    URL of a SSL certificate associated with this target proxy. Can and must be used only with https type proxy.
+
+    *type:* string
+    *default:*
+  * `additional_settings`
+    Additional setting for target proxy
+
+    *default:* {}
+  * `name`
+    Optional target proxy name. By default it will be target proxy id.
+
+    *type:* string
+    *default:*
+  * `target_proxy_type`
+    This field indicates if this target proxy is a HTTP or HTTPS based target proxy. Possible values are: 'http' and 'https'.
+
+    *type:* string
+    *default:* http
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `url_map`
+    URL of a URL map which specifies how traffic from this target proxy should be redirected.
+
+    *type:* string
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+
+
+
+
+
+## cloudify.gcp.nodes.UrlMap
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+Maps URLs to BackendServices
+
+
+
+**Properties:**
+
+
+  * `default_service`
+    URL of a backend service to which this URL map will redirect traffic by default.
+
+    *type:* string
+    *default:*
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `additional_settings`
+    Additional setting for url map
+
+    *default:* {}
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+  * `name`
+    Optional health check name. By default it will be URL map id.
+
+    *type:* string
+    *default:*
+
+
+
+
+
+## cloudify.gcp.nodes.Volume
+**Derived From:** [cloudify.nodes.Volume]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP Volume.
+
+A virtual disk which can be attached to Instances.
+
+
+
+**Properties:**
+
+
+  * `additional_settings`
+    Additional setting for volume
+
+    *default:* {}
+  * `name`
+    Optional disk name. By default it will be disk id.
+
+    *type:* string
+    *default:*
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the Google Cloud Platform API.
+
+    *default:* {}
+  * `image`
+    The image of the Volume.
+
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource.
```

Comment:
Indicate**s** whether the resource exists **and should be used (true),** or if Cloudify should create the resource **(false).**